<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestpointWidget</class>
 <widget class="QWidget" name="TestpointWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>112</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="admittanceLabel">
     <property name="text">
      <string>Admittance:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="phLabel">
     <property name="text">
      <string>pH:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="orpLabel">
     <property name="text">
      <string>ORP:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="temperatureLabel">
     <property name="text">
      <string>Temperature:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="voltammetryLabel">
     <property name="text">
      <string>Voltammetry:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="admittance">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="phValue">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="orpValue">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="temperature">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="voltammetryValues">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
